idd/Energy+.idd.in

```diff
@@ -36464,6 +36464,776 @@ ZoneHVAC:Baseboard:RadiantConvective:Electric,
        \minimum 0
        \maximum 1
 
+ZoneHVAC:CoolingPanel:RadiantConvective:Water,
+  A1 , \field Name
+       \required-field
+  A2 , \field Available Schedule Name
+       \required-field
+       \type object-list
+       \object-list ScheduleNames
+  A3 , \field Inlet Node Name
+       \required-field
+  A4 , \field Outlet Node Name
+       \required-field
+  N1 , \field Rated Inlet Water Temperature
+       \type real
+       \units C
+       \default 5.0
+  N2 , \field Rated Inlet Space Temperature
+       \type real
+       \units C
+       \default 24.0
+  N3 , \field Rated Water Mass Flow Rate
+       \type real
+       \minimum> 0.0
+       \units Kg/s
+       \default 0.063
+  N4 , \field Rated Capacity 
+       \autosizable
+       \type real
+       \units W
+  N5 , \field Maximum Chilled Water Flow Rate 
+       \required-field
+       \autosizable
+       \type real
+       \units m3/s
+  A5 , \field Control Type
+       \note (Temperature on which unit is controlled)
+       \type choice
+       \key MeanAirTemperature
+       \key MeanRadiantTemperature
+       \key OperativeTemperature
+       \key OutdoorDryBulbTemperature
+       \key OutdoorWetBulbTemperature
+       \default MeanAirTemperature
+  N6 , \field Cooling Control Throttling Range
+       \units deltaC
+       \minimum 0.5
+       \default 0.5
+  A6 , \field Cooling Control Temperature Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+  A7 , \field Condensation Control Type
+       \type choice
+       \key Off
+       \key SimpleOff
+       \key VariableOff
+       \default SimpleOff
+  N7 , \field Condensation Control Dewpoint Offset
+       \units C
+       \default 1.0
+  N8 , \field Fraction Radiant
+       \required-field
+       \type real
+       \minimum 0
+       \maximum 1
+  N9 , \field Fraction of Radiant Energy Incident on People
```

Comment:
@Myoldmopar Well, not really because it will depend on the placement of the panel with respect to where people are at.  One would think that these would have a placement such that they would have a significant impact on the people in the space.  However, what exactly this number is will vary with each building.  So, putting a default in is really just picking a random number out of a hat...at least in my opinion.  Do you still want me to put a number in?